src/guide/jbs-jdk-bug-system.md

```diff
@@ -163,9 +164,10 @@ Now that the issue is in the right component and has the basic information, the
    * An approach that has been used for getting a consistent view of priority is to consider three aspects of the issue: **Impact** of the issue; **Likelihood** of it occurring; and, whether there is a **Workaround**. The higher the impact and likelihood the higher the priority; then, having a workaround reduces that priority - but mostly where the impact and likelihood aren't that severe.
 1. Ensure the [Affects Version/s]{.jbs-field} field is correct (within reason).
    * This may involve reproducing the bug, if doing so is fast and easy.
-   * In addition to the version where the bug was found, take special care to also investigate if the bug affects the generally supported releases - e.g. the latest "LTS" release and the latest six-month release.
+   * In addition to the version where the bug was found, take special care to also investigate if the bug affects the mainline.
```

Comment:
I see "mainline" as a name rater than a thing, so I wouldn't have "the" in front of it in this sentence.

Reply:
yes good point, makes sense - have removed the offending "the"